iidm/iidm-serde/src/test/resources/V1_0/tieline.xml

```diff
@@ -37,6 +37,6 @@
             </iidm:ratioTapChanger>
         </iidm:twoWindingsTransformer>
     </iidm:substation>
-    <iidm:tieLine id="NHV1_NHV2_1" ucteXnodeCode="X1" bus1="NHV1" connectableBus1="NHV1" voltageLevelId1="VLHV1" bus2="NHV2" connectableBus2="NHV2" voltageLevelId2="VLHV2" p1="302.4440612792969" q1="98.74027252197266" p2="-300.43389892578125" q2="-137.18849182128906" id_1="NHV1_NHV2_1.1" r_1="1.5" x_1="16.5" g1_1="0.0" b1_1="9.65E-5" g2_1="0.0" b2_1="9.65E-5" xnodeP_1="-301.47434650169697" xnodeQ_1="-118.8505833038079" id_2="NHV1_NHV2_1.2" r_2="1.5" x_2="16.5" g1_2="0.0" b1_2="9.65E-5" g2_2="0.0" b2_2="9.65E-5" xnodeP_2="301.472319625184" xnodeQ_2="118.76748245598334"/>
-    <iidm:tieLine id="NHV1_NHV2_2" ucteXnodeCode="X2" bus1="NHV1" connectableBus1="NHV1" voltageLevelId1="VLHV1" bus2="NHV2" connectableBus2="NHV2" voltageLevelId2="VLHV2" p1="302.4440612792969" q1="98.74027252197266" p2="-300.43389892578125" q2="-137.18849182128906" id_1="NHV1_NHV2_2.1" r_1="1.5" x_1="16.5" g1_1="0.0" b1_1="9.65E-5" g2_1="0.0" b2_1="9.65E-5" xnodeP_1="-301.47434650169697" xnodeQ_1="-118.8505833038079" id_2="NHV1_NHV2_2.2" r_2="1.5" x_2="16.5" g1_2="0.0" b1_2="9.65E-5" g2_2="0.0" b2_2="9.65E-5" xnodeP_2="301.472319625184" xnodeQ_2="118.76748245598334"/>
+    <iidm:tieLine id="NHV1_NHV2_1" ucteXnodeCode="X1" bus1="NHV1" connectableBus1="NHV1" voltageLevelId1="VLHV1" bus2="NHV2" connectableBus2="NHV2" voltageLevelId2="VLHV2" p1="302.4440612792969" q1="98.74027252197266" p2="-300.43389892578125" q2="-137.18849182128906" id_1="NHV1_NHV2_1.1" r_1="1.5" x_1="16.5" g1_1="0.0" b1_1="9.65E-5" g2_1="0.0" b2_1="9.65E-5" xnodeP_1="-301.43898434342725" xnodeQ_1="-118.89616773372641" id_2="NHV1_NHV2_1.2" r_2="1.5" x_2="16.5" g1_2="0.0" b1_2="9.65E-5" g2_2="0.0" b2_2="9.65E-5" xnodeP_2="301.43897577958876" xnodeQ_2="118.89616087695104"/>
+    <iidm:tieLine id="NHV1_NHV2_2" ucteXnodeCode="X2" bus1="NHV1" connectableBus1="NHV1" voltageLevelId1="VLHV1" bus2="NHV2" connectableBus2="NHV2" voltageLevelId2="VLHV2" p1="302.4440612792969" q1="98.74027252197266" p2="-300.43389892578125" q2="-137.18849182128906" id_1="NHV1_NHV2_2.1" r_1="1.5" x_1="16.5" g1_1="0.0" b1_1="9.65E-5" g2_1="0.0" b2_1="9.65E-5" xnodeP_1="-301.43898434342725" xnodeQ_1="-118.89616773372641" id_2="NHV1_NHV2_2.2" r_2="1.5" x_2="16.5" g1_2="0.0" b1_2="9.65E-5" g2_2="0.0" b2_2="9.65E-5" xnodeP_2="301.43897577958876" xnodeQ_2="118.89616087695104"/>
```

Comment:
here we can see nicely the bug fix, it seems this network has a good powerflow solution (bus voltages and angles), and we see here it had bad (unbalanced) boundary data, which are now fixed.

**Before** the fix:
- the difference boundary P was xnodeP_1="-301.47434650169697" vs xnodeP_2="301.472319625184", hence 2 digits precision.
- the difference boundary Q was xnodeQ_1="-118.8505833038079" vs xnodeQ_2="118.76748245598334", hence 0 (!) digits precision.

**After** the fix:
- on P: xnodeP_1="-301.43898434342725" and xnodeP_2="301.43897577958876" hence 4 digits precision (+2 !)
- on Q: xnodeQ_1="-118.89616773372641" and xnodeQ_2="118.89616087695104" hence 5 digits precision (+3 !)